Gateway/Response/Card/PaymentDetailsHandler.php

```diff
@@ -45,7 +43,7 @@ public function handle(array $handlingSubject, array $response): void
             $this->setTransactionDataToPayment($payment, $response);
             $this->setAdditionalTransactionData($payment, $response);
 
-            $payment->getOrder()->setCanSendNewEmailFlag(false);
+            $payment->getOrder()->setCanSendNewEmailFlag(true);
```

Comment:
What exactly is the question here I don't understand?

Reply:
Seeing a boolean flag switch from `false` to `true` sets off alarm bells for me without context, in case it was an accidental change rather than fixing a bug/enabling new functionality.

So my question is, can you confirm that this is an intentional and correct change?

Reply:
I can confirm it was intentional and was done by me. As for why it was initially false, I cannot answer as I did not initially write this piece of code.

I can confirm however that it needs to be true for card payments and false for hosted / pay by bank payment.